docs/development/core-development.asciidoc

```diff
@@ -10,3 +10,5 @@ include::core/development-basepath.asciidoc[]
 include::core/development-dependencies.asciidoc[]
 
 include::core/development-modules.asciidoc[]
+
+include::plugin/development-elasticsearch.asciidoc[]
```

Comment:
We should add a link to **Communicating with elasticsearch** like the rest of the headings have.